sources/packages/web/src/views/Login.vue

```diff
@@ -33,7 +33,7 @@ export default class Login extends Vue {
   login() {
     AppConfigService.shared.authService?.login({
       idpHint: "bcsc",
-      redirectUri: `${location.origin}/`
+      redirectUri: `${location.origin}/student`
```

Comment:
I believe that the specific redirect to the "/student" is no longer needed.
The "created" hook on App.vue will take care of it, as per my understanding.

Reply:
Yes, app should redirect to root url and from there based on authentication state of KeyCloak App.vue will redirect them to appropriate page.

Reply:
Fixed